### PR TITLE
[13.0][MIG] survey_resource_booking: Migration to 13.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 13.0

Please @chienandalu and @pedrobaeza can you review it?

@Tecnativa TT31325

### CODE REVIEW

File: survey_resource_booking/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Jairo Llopis
```
Comment (pedrobaeza) [inline survey_resource_booking/readme/CONTRIBUTORS.rst:3]: Put yourself here

File: test-requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+cssselect
```
Comment (pedrobaeza) [inline test-requirements.txt:1]: Why is this needed?

**Review by pedrobaeza — APPROVED** [review]
Tested on runboat and code review

File: test-requirements.txt (hunk 2 above)
Comment (victoralmau) [reply test-requirements.txt:1]: It is a dependency of the `resource_booking` addon that is only used in tests: https://github.com/OCA/calendar/blob/13.0/test-requirements.txt

**Review by chienandalu — APPROVED** [review]
